cmd/delete.go

```diff
@@ -84,24 +86,30 @@ var deleteCmd = &cobra.Command{
 
 		return c.Run(objs)
 	},
-	Long: `Delete Kubernetes resources from a cluster, as described in the local
-configuration.
+	Long: `
+The ` + "`delete`" + ` command removes Kubernetes resources from a cluster, as described
```

Comment:
Slight rewording ...

Instead of `The delete command removes Kubernetes resources from a cluster, as described in local component manifests`,

How about: `The delete command removes Kubernetes resources, as described in local component manifests, from a cluster`.

Because the first seems to be saying that the local manifests are describing the delete command.